specs/ClearBrowsingData.md

```diff
@@ -247,22 +254,24 @@ namespace Microsoft.Web.WebView2.Core
 {
     [Flags] enum CoreWebView2BrowsingDataKinds
     {
-        AppCache = 1,
-        FileSystems = 2,
-        IndexedDb = 4,
-        LocalStorage = 8,
-        WebSql = 16,
-        CacheStorage = 32,
-        AllDomStorage = 64,
-        Cookies = 128,
-        AllSite = 256,
-        DiskCache = 512,
-        DownloadHistory = 1024,
-        GeneralAutofill = 2048,
-        PasswordAutosave = 4096,
-        BrowsingHistory = 8192,
-        Settings = 16384,
-        AllProfile = 32768,
+        /// [ObsoleteAttribute("This property is obsolete. Do not use it any more.", true)]
+        /// AppCache = 1,
```

Comment:
This cannot be removed via comment. The enum value will need to remain and will be documented as deprecated.

Reply:
Will update later.

Reply:
Have never been shipped, and remove it directly.